app/models/policy.rb

```diff
@@ -81,6 +93,13 @@ def destroy_orphaned_business_objective
 
   private
 
+  def unassigned_minor_versions
+    stored_versions = profiles.pluck(:os_minor_version)
+    Host.os_minor_versions(hosts.pluck(:id)).reject do |version|
+      stored_versions.include?(version) # Ignore already stored minor versions
```

Comment:
`stored_versions` is a list of strings (i.e. `["9", ""]`), but `version` is an integer:

```suggestion
      stored_versions.include?(version.to_s) # Ignore already stored minor versions
```